src/lib/Hydra/Controller/Project.pm

```diff
@@ -169,6 +169,8 @@ sub updateProject {
             , emailoverride => ""
             , triggertime => time
             });
+    } else {
+        $project->jobsets->search({ name => ".jobsets" })->delete;
```

Comment:
Should we maybe also clear the `decltype` and `declvalue`?

Reply:
I opted to do so, but it can be easily reverted.

Reply:
I think this is a good idea.